packages/@sanity/desk-tool/src/panes/documentPane/timeline/timelineItem.styled.tsx

```diff
@@ -0,0 +1,103 @@
+import {Text, Box, MenuItem, Theme, Flex} from '@sanity/ui'
+import styled, {css} from 'styled-components'
+import {TimelineItemState} from './types'
+
+export interface IconWrapperProps {
+  theme: Theme
+}
+
+export interface TimelineItemProps {
+  state: TimelineItemState
+  theme: Theme
+  isHovered: boolean
+}
+
+export const IconWrapper = styled(Flex)(({theme}: IconWrapperProps) => {
+  const borderColor = theme.sanity.color.base.skeleton?.from
+
+  return css`
+    --timeline-hairline-width: 1.3px;
```

Comment:
The behaviour of fraction pixels varies. In Chrome it is truncated or rounded down, and other browsers does it in slight different ways.

Reply:
Hello 8) This is a part of the code that was already there once I jumped in it. That being said, do you feel that it would be safer to move it to 1px or to 2px as to not have that possible variation? :)

Reply:
I think it in this case is rounded down to 1px. I think a "hairline" is 1px in Sanity UI. What do you think @mariuslundgard ?

Reply:
As spoken in the SU, changed it from 1.3 to 1 :)